scripts/common.sh.in

```diff
@@ -213,9 +218,24 @@ run() {
     fi
 }
 
-die() {
-    echo 1>&2 "$@"
-    return 1
+run_with_retry() {
+    local retries=$1
+    local delay=$2
+    shift 2
+
+    local -i i=0
+    while true; do
+        if [[ $(( ++i )) -gt $retries ]]; then
+            die "Failed to run '$*' after ${retries} retries."
+        fi
+
+        echo -n "> ${i}/${retries} "
```

Comment:
I have a question about this, out of curiosity, 
if this passes in one try most of the times, then do we want to print the attempts before or after the attempt is done? 
in one of these options we can keep virtually the same log in nominal case and only print retry information when the first attempt fails.

Reply:
Hello @eg-ayoub !!!

Good catch!!!

I really did not ask anyone before to do that. I implemented that way based in my passed experiences.

Do you think it will cause a negative impact to customers or to customer experience and support teams ?

If yes, I will change. If not, I'll keep this way.